### PR TITLE
refactoring libraries folder

### PR DESCRIPTION
**Pull request by sdk-banuba**

### CODE REVIEW

File: libraries/renderer/render_thread.cpp [hunk 1]
```diff
@@ -45,4 +51,5 @@ namespace bnb::render
         m_renderer.reset();
         glfwMakeContextCurrent(nullptr);
     }
-} // bnb::render
+
+} /* namespace bnb::render */
```
Comment (dmitrystoyanovsky) [inline libraries/renderer/render_thread.cpp:55]: Please apply the style of Banuba SDK by applying ./scripts/reformat.py --docker
- Copy code to the Banuba SDK folder
- Update reformat script to process copied folder
- Run script
- Copy changed files back.

Comment (dmitrystoyanovsky) [reply libraries/renderer/render_thread.cpp:55]: To mark the end of namespace '}' it is used the // comment style (one line)

File: libraries/renderer/renderer.cpp [hunk 2]
```diff
@@ -1,18 +1,17 @@
 #include "renderer.hpp"
 #include "opengl.hpp"
 
-// RGBA texture
```
Comment (dmitrystoyanovsky) [inline libraries/renderer/renderer.cpp:4]: Please recheck that change - it seems you made the wrong one.

File: libraries/renderer/renderer.cpp [hunk 3]
```diff
@@ -1,18 +1,17 @@
...
+// NV12
 namespace
 {
-    // NOTE: The shader below assumes that texture image is oriented head at the top, so it inverts y-axis
```
Comment (dmitrystoyanovsky) [inline libraries/renderer/renderer.cpp:7]: Wrong change - recheck

File: libraries/utils/glfw_utils/glfw_window.cpp [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+#include <bnb/effect_player/utility.hpp>
+#include <bnb/utils/defs.hpp>
+
+#include <glad.h>
+
+namespace bnb::gl
+{
+
+    /* glfw_window::glfw_window */
```
Comment (dmitrystoyanovsky) [inline libraries/utils/glfw_utils/glfw_window.cpp:11]: I think such comments are excessive, please reconsider their usage - I'd prefer to remove them to make the style closer to SDK.

**Review by dmitrystoyanovsky — CHANGES_REQUESTED** [review]

**Comment by sdk-banuba** [comment]
this PR is incorrect, based on incorrect branch

**Pull request closed without merging**